WinUIGallery/CompactSizePage.xaml

```diff
@@ -1,15 +0,0 @@
-﻿<Page
-    x:Class="AppUIBasics.CompactSizePage"
```

Comment:
I assume compact sizing is not something WinUI 3 is encouraging to use?

Reply:
Or because it's also under `WinUIGallery\ControlPages` ?